vllm/platforms/interface.py

```diff
@@ -143,6 +143,9 @@ def is_cuda_alike(self) -> bool:
         """Stateless version of :func:`torch.cuda.is_available`."""
         return self._enum in (PlatformEnum.CUDA, PlatformEnum.ROCM)
 
+    def is_rocm_aiter_linear_enabled(self) -> bool:
```

Comment:
Same comment as the other AITER kernels. Let's remove these checks from the platform classes.